handlers/resolveDataObjectUri.js

```diff
@@ -0,0 +1,8 @@
+const { dataObjectUriToHttps, Response, promiseHandler } = require('../common/helpers');
+
+const resolveDataObjectUri = promiseHandler(async (req) => {
+    const { uri } = req.body || {};
+    return new Response(200, { url: dataObjectUriToHttps(uri) });
+});
+
+module.exports = resolveDataObjectUri;
```

Comment:
What's the reason for adding this endpoint?  Is Terra wanting to call and parse the resulting DOS object itself?

Reply:
Yeah, I think we want to do as much as we can in the UI.

Reply:
For public data, it shouldn't really be an issue, but for private data that requires Bond to get key information, you'll need to have logic implemented to know how to talk to Bond and which provider to talk to, etc.  I think the goal of Martha was to try to isolate this logic.  I will invoke the @dvoet to make sure he's aware.

Reply:
Centralizing that logic in Martha continues to be a goal. See `getSignedUrl` above.

Reply:
spreading the logic to parse the json response from a dos/drs resolution is something martha aims to avoid. The spec is unstable and we want to be able to handle multiple versions of the spec in 1 place, which is martha. Exposing this url goes against martha's purpose.

Reply:
What's the abstraction we want to provide, then? DOS => metadata+gsuri?